src/CSharp/CodeCracker/CodeCracker.csproj

```diff
@@ -36,6 +36,8 @@
     <Compile Include="Performance\StringBuiderInLoopCodeFixProvider.cs" />
     <Compile Include="Performance\StringBuilderInLoopAnalyzer.cs" />
     <Compile Include="Refactoring\AllowMembersOrderingAnalyzer.cs" />
+    <Compile Include="Refactoring\AllowMembersOrderingCodeFixProvider.StyleCop.cs" />
+    <Compile Include="Refactoring\AllowMembersOrderingCodeFixProvider.Base.cs" />
```

Comment:
Instead of create the file with the same name of the class, I chose to use the AllowMembersOrderingCodeFixProvider as a prefix to make easy to locate each implementation of BaseAllowMembersOrderingCodeFixProvider